packages/wasm-terminal/lib/command-runner/command-runner.ts

```diff
@@ -225,14 +236,20 @@ export default class CommandRunner {
   }
 
   async _spawnProcessAsService(commandOptionIndex: number) {
+    // Get our filesystem state
+    const wasmFsJson = this.wasmTerminalConfig.wasmFs.toJSON();
+
     const process = new Process(
+      // Command Options
       this.commandOptionsForProcessesToRun[commandOptionIndex],
+      // WasmFs File System JSON
+      wasmFsJson,
       // Data Callback
       this._processDataCallback.bind(this, commandOptionIndex),
       // End Callback
-      this._processEndCallback.bind(this, commandOptionIndex),
+      this._processEndCallback.bind(this, { commandOptionIndex }),
```

Comment:
So I remember doing this, are you sure you are up to date with master? 🤔

Reply:
Yeah, this is from your commits :)